channeldb/kvdb/etcd/embed.go

```diff
@@ -42,7 +42,7 @@ func NewEmbeddedEtcdInstance(path string) (*BackendConfig, func(), error) {
 	cfg.Dir = path
 
 	// To ensure that we can submit large transactions.
-	cfg.MaxTxnOps = 8192
+	cfg.MaxTxnOps = 16384
```

Comment:
oh, where do we hit this?

And do we check that somewhere that the max op num is set large enough when using the non-embedded version?